web/src/stories/docs/about/Intro.story.mdx

```diff
@@ -1,14 +1,64 @@
 import { Meta } from '@storybook/addon-docs/blocks';
 
 <Meta
-  title="Docs/Getting Started/Installation"
+  title="Docs/Intro"
   parameters={{
     viewMode: 'docs',
     previewTabs: { canvas: { hidden: true } },
     canvas: { disable: true },
   }}
 />
 
+<div style={{ margin: '0 auto', maxWidth: '600px', color: 'white', textAlign: 'center' }}>
+  <img
+    style={{ width: '75%', margin: '0 auto 2em auto' }}
+    src="/assets/octant-logo.png"
+  />
+  <br />
+  <br />
+
+
+![Octant demo](/assets/octant-demo.gif)
+
+<br/>
+
+<a href="https://github.com/vmware-tanzu/octant/releases/latest">
+  <img src="https://img.shields.io/github/release/vmware-tanzu/octant.svg" />
+</a>
+
+&nbsp;&nbsp;
+<a href="https://github.com/vmware-tanzu/octant">
+  <img src="https://github.com/vmware-tanzu/octant/workflows/preflight-checks/badge.svg" />
+</a>
+&nbsp;&nbsp;
+<a href="https://github.com/vmware-tanzu/octant/blob/master/LICENSE">
+  <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" />
+</a>
+&nbsp;&nbsp;
+<a href="https://github.com/vmware-tanzu/octant">
+  <img
+    alt="GitHub stars"
+    src="https://img.shields.io/github/stars/vmware-tanzu/octant?style=social"
+  />
+</a>
+&nbsp;&nbsp;
+<a href="https://bestpractices.coreinfrastructure.org/projects/4142">
+  <img
+    alt="Core infrastructure best practices"
+    src="https://bestpractices.coreinfrastructure.org/projects/4142/badge"
+  />
+</a>
+
+---
+
+<h3>A highly extensible platform for developers to better understand the complexity of Kubernetes clusters.</h3>
+<p>
+Octant is a tool for developers to understand how applications run on a Kubernetes cluster.
+It aims to be part of the developer's toolkit for gaining insight and approaching complexity found in Kubernetes.
+Octant offers a combination of introspective tooling, cluster navigation, and object management along with a plugin system to further extend its capabilities.
+</p>
+</div>
+
 # Getting Started
```

Comment:
Shouldn't `Getting Started` be a separate page?

Reply:
I merged the Logo and Getting started pages so that there was some call to action on the main landing page. Similar to how we have a README in GitHub, I found that having the prompt for getting started in the Intro page felt better.

Reply:
That's cool, but I almost missed it because it was opened inside a small browser window. It should be allright with elimination of the white space at the top of the page.